modules/sbo-creating-a-postgresql-database-instance-power-z.adoc

```diff
@@ -45,53 +34,31 @@ EOD
 ----
 +
 The annotations added in this `Database` CR enable the service binding connection and trigger the Operator reconciliation.
-+
-The output verifies that the database instance is created:
+
+.Verification
+
+. The output verifies that the database instance is created:
 +
 .Example output
 [source,terminal]
 ----
 database.postgresql.dev4devs.com/sampledatabase created
 ----
 
-. After you have created the database instance, ensure that all the pods in the `my-postgresql` namespace are running (it will take a few minutes):
+. After you have created the database instance, ensure that all the pods in the `my-petclinic` namespace are running:
 +
 [source,terminal]
 ----
-$ oc get pods -n my-postgresql
+$ oc get pods -n my-petclinic
 ----
 +
-The output verifies that the database is created:
+The output, which takes a few minutes to display, verifies that the database is created and configured:
 +
 .Example output
 [source,terminal]
 ----
 NAME                                     READY    STATUS      RESTARTS   AGE
 sampledatabase-cbc655488-74kss            0/1     Running        0       32s
 ----
-+
-
-The new database is empty at this stage. You can set its schema and project a sample data set to interact with the sample application.
-
-. Initialize the database with the schema and sample data. To do so, use the following custom shell script by copying the code into the shell and running it:
-+
-[source,terminal]
-----
-$ cat << EOD | bash
-#!/bin/bash
-
-export pgo_cluster_name=sampledb
-export cluster_namespace=my-postgresql
-export pgo_cluster_username=sampleuser
-nohup oc -n "\${cluster_namespace}" port-forward svc/sampledatabase 5432:5432 &
-sleep 5
-curl -LO https://raw.githubusercontent.com/spring-petclinic/spring-petclinic-rest/master/src/main/resources/db/postgresql/initDB.sql
-psql -h localhost -U "\${pgo_cluster_username}" "\${pgo_cluster_name}" -f initDB.sql
-curl -LO https://raw.githubusercontent.com/spring-petclinic/spring-petclinic-rest/master/src/main/resources/db/postgresql/populateDB.sql
-psql -h localhost -U "\${pgo_cluster_username}" "\${pgo_cluster_name}" -f populateDB.sql
-EOD
-----
-
-The output in the terminal shows you that the database is being configured for the application.
```

Comment:
Need this to populate the DB with the sample data.

Reply:
with the new app, there is no need for populating the DB with the sample data, the application should initialize the DB o its own.